_data/hosting.yml

```diff
@@ -246,21 +263,4 @@ websites:
       img: wpengine.png
       tfa: Yes
       software: Yes
-      doc: https://wpengine.com/support/2-factor-authentication/
-
-    - name: Ukraine
-      url: https://www.ukraine.com.ua/
-      img: ukraine.png
-      tfa: Yes
-      sms: Yes
-      phone: Yes
-      software: Yes
-      hardware: Yes
-      doc: https://www.ukraine.com.ua/faq/rekomendatsii-po-zashchite-ot-vzloma-uchetnoj-zapisi.html
-
-    - name: Easy
-      url: https://easy.gr/
-      img: easy.png
-      tfa: Yes
-      software: Yes
-      doc: https://easy.gr/el/faq
+      doc: https://wpengine.com/support/2-factor-authentication/
```

Comment:
As per our `.editorconfig`, it contains `insert_final_newline = true`.
Please respect that.